docs/USAGE-GUIDE.md

```diff
@@ -1260,6 +1260,16 @@ handshake.
 
 **s2n_connection_is_session_resumed** returns 1 if the handshake was abbreviated, otherwise returns 0.
 
+### s2n\_connection\_get\_selected\_cert
+
+```c
+struct s2n_cert_chain_and_key s2n_connection_get_selected_cert(struct s2n_connection *conn);
+```
+
+Return the certificate that was used during the TLS handshake. If **conn** is a server connection, the certificate selected will depend on the
```

Comment:
We could call out the client conn behavior here too. I think it will return the client cert after the CertificateRequest has been received. Theoretically this function will also be useful if our client starts obeying any sig-hash algs in the CertificateRequest. 

I'm torn on this, but since there are potentially two certificates in the mix (client and server/peer and own) we could also change the name to include "my" or "own" (s2n_connection_get_own_selected_cert) to make it more clear what this function is returning?

Reply:
updated description to include how client mode is treated